2/FDE/domandeElettronica.md

```diff
@@ -0,0 +1,60 @@
+## Domande (COVID-19 branch)
+
```

Comment:
Il nome del file sarebbe carino metterlo in camelCase come gli altri, in questo caso ad esempio domandeElettronica.md